frontends/precinct-scanner/src/screens/election_manager_screen.tsx

```diff
@@ -149,36 +156,53 @@ export function ElectionManagerScreen({
     <ScreenMainCenterChild infoBarMode="admin">
       <Prose textCenter>
         <h1>Election Manager Settings</h1>
-        <p>
-          <Select
-            id="selectPrecinct"
-            data-testid="selectPrecinct"
-            value={precinctSelectionValue}
-            onBlur={changeAppPrecinctSelection}
-            onChange={changeAppPrecinctSelection}
-            large
-          >
-            <option value="" disabled>
-              {SELECT_PRECINCT_TEXT}
-            </option>
-            {election.precincts.length > 1 && (
-              <option value={ALL_PRECINCTS_OPTION_VALUE}>
-                {ALL_PRECINCTS_NAME}
+        {pollsState === 'polls_closed_initial' ? (
+          <p>
+            <Select
+              id="selectPrecinct"
+              data-testid="selectPrecinct"
+              value={precinctSelectionValue}
+              onBlur={handlePrecinctSelectionChange}
+              onChange={handlePrecinctSelectionChange}
+              disabled={scannerStatus.ballotsCounted > 0}
+              large
+            >
+              <option value="" disabled>
+                {SELECT_PRECINCT_TEXT}
               </option>
-            )}
-            {[...election.precincts]
-              .sort((a, b) =>
-                a.name.localeCompare(b.name, undefined, {
-                  ignorePunctuation: true,
-                })
-              )
-              .map((precinct) => (
-                <option key={precinct.id} value={precinct.id}>
-                  {precinct.name}
+              {election.precincts.length > 1 && (
+                <option value={ALL_PRECINCTS_OPTION_VALUE}>
+                  {ALL_PRECINCTS_NAME}
                 </option>
-              ))}
-          </Select>
-        </p>
+              )}
+              {[...election.precincts]
+                .sort((a, b) =>
+                  a.name.localeCompare(b.name, undefined, {
+                    ignorePunctuation: true,
+                  })
+                )
+                .map((precinct) => (
+                  <option key={precinct.id} value={precinct.id}>
+                    {precinct.name}
+                  </option>
+                ))}
+            </Select>
+          </p>
+        ) : (
+          precinctSelection && (
+            <p>
+              <ChangePrecinctButton
```

Comment:
Is it possible to reuse this component for the initial selection of a precinct as well? It feels like we are duplicating a lot in this component and in the code above here.

Reply:
Could be, I think we might want to reuse it in the BMD too. I will take a look.

Reply:
Refactored.